### DISCUSSION TITLE
[Suggestion] Add show inventory/enderchest into CMI chat

### DISCUSSION THREAD

**Discussion by CactusTheGuy** [Zrips/CMI#8647] [Feature Requests]
add the ability to show inventory or ender chest either in public chat or private chat

**Comment by Zrips** [comment]
Public chat would be hammered with so much information at once. We can't really use only one hover to display all items, so it maybe would be one hover for each item with its information like we have with [item], but it would create quite a big message

**Comment by CactusTheGuy** [comment]
If that the case
Could it possibly open a inventory GUI that allow to view the inventory/ender chest of the sender player ?

**Reply by CactusTheGuy** [reply]
And after 5 mins, the plugin should remove that inventory/ender chest view off the memory
